pygem/bffd.py

```diff
@@ -0,0 +1,58 @@
+from pygem.cffd import CFFD
+import numpy as np
+
+"""
+Utilities for performing Barycenter Free Form Deformation (BFFD).
+
+:Theoretical Insight:
+    It performs Free Form Deformation while trying to enforce the barycenter to be a certain value specified by the user.
+    The constraint is enforced exactly (up to numerical errors).
+    For details see the mother and the grandmother clases.
+
+ 
+"""
+
+
+
+class BFFD(CFFD):
+    '''
+    Class that handles the Barycenter Free Form Deformation on the mesh points.
+ 
```

Comment:
Arguments doc is missing

Reply:
Hi. For this class I did not add any argument doc because they are exactly the same as the mother class. The only difference is that in the child class a mother class member has a default value. I should add them anyway? 
Thanks for your comments.

Reply:
Yes please. So the online documentation will be self-contained.